### ISSUE TITLE
Error after upgrade webpack

### ISSUE THREAD

**Issue by OzzyCzech** [jantimon/html-webpack-harddisk-plugin#23]
Hi there,

I am getting follow warning after upgrade `webpack`

```txt
warning " > html-webpack-harddisk-plugin@1.0.1" has incorrect peer dependency "html-webpack-plugin@^2.0.0 || ^3.0.0".
```

and from `yarn check`

```txt
"html-webpack-harddisk-plugin#html-webpack-plugin@^2.0.0 || ^3.0.0" doesn't satisfy found match of "html-webpack-plugin@4.0.1"
```

**Comment by glen-84** [comment]
This will happen after upgrading `html-webpack-plugin`, not `webpack`.

`peerDependencies` have been [updated](https://github.com/jantimon/html-webpack-harddisk-plugin/commit/d7624ef2881aec6ff901ada2c823a872ca6f3124), but there hasn't been a new release since then.

@jantimon Will you be releasing a new version soon?

**Comment by OzzyCzech** [comment]
thanks

**Comment by jantimon** [comment]
released as html-webpack-harddisk-plugin 2.0
